### ISSUE TITLE
Question about SemanticKITTI→SemanticSTF Evaluation

### ISSUE THREAD

**Issue by at7am** [xiaoaoran/SemanticSTF#1]
Hi, and thanks for sharing the good work! 

I had a general question regarding the evaluation setup that you show in Table 2 of the paper. 
The question is, when you train your model on the SemanticKITTI, you classify points in one of the 19 classes (car, ..., traf.). However, the SemanticSTF has also the 4 adverse weather classes in addition to the 19 base classes.  
How do you segment them if your model outputs only 19 logits?  

The same goes of the SynLiDAR→SemanticSTF evaluation.

**Comment by xiaoaoran** [comment]
Hi, @at7am 

We separate data according to weather type and evaluate miou. Please refer to 'evaluate_by_weather.py'

**Comment by at7am** [comment]
Hi, thanks for your answer. 
So basically the table shows the class-wise IoU for the entire dataset, and then the mIoU when segmenting in a particular weather condition?

**Comment by xiaoaoran** [comment]
Yes.
